lib/property_sets/active_record_extension.rb

```diff
@@ -175,6 +175,12 @@ def update!(attributes)
       end
       alias update_attributes! update!
 
+      def update_columns(attributes)
+        attrs = attributes.reject{|k,_| self.class.delegated_property_set_attributes.include?(k) }
+        raise ArgumentError, "Empty list of attributes to change" if attrs.empty?
```

Comment:
Is it really an error if an update doesn't change the property set attributes?

Reply:
I'm glad you asked... Rails 5.2 and above let the empty set go through but eventually hit an error. Below 5.2 and they handle that ArgumentError. So I just copied the behavior.
An alternative approach would be to have 2 different styles of testing for the different rails/activerecord versions.